Add tag cardinality common field (#157)
DataDog/php-datadogstatsd: README.md

After the client is created, you can start sending Service Checks to Datadog. See the dedicated [Service Check Submission: DogStatsD documentation](https://docs.datadoghq.com/developers/service_checks/dogstatsd_service_checks_submission/?code-lang=php) to see how to submit a Service Check to Datadog.

### Origin detection in Kubernetes

> Origin detection is a method to detect which pod DogStatsD packets are coming from in order to add the pod's tags to the tag list.

Reviewer: ```suggestion
Origin detection is a method to detect the pod that DogStatsD packets are coming from and add the pod's tags to the tag list.
```
